packages/astro/src/runtime/client/dev-overlay/ui-library/tooltip.ts

```diff
@@ -57,6 +57,11 @@ export class DevOverlayTooltip extends HTMLElement {
 				padding: 8px;
 			}
 
+			.section-content {
+				max-height: 250px;
+    		overflow-y: scroll;
+			}
```

Comment:
The virtual Playwright window is quite small, and somehow in some cases it couldn't see the tooltip. This is generally just a good change, because the tooltip could get quite long (ex: the docs sidebars)